backend/back4app-cloud-code/back4app-cloud-code/CheckAnswer.js

```diff
@@ -0,0 +1,93 @@
+class CheckAnswerEngine {
+
+  // checks the correctness of a given answer by comparing to a list of acceptable answers
+  static checkAnswerList(givenAnswer, acceptableAnswers) {
+    let answerChecks = acceptableAnswers.map((ans) => (this.checkAnswer(givenAnswer, ans)));
+    return answerChecks.reduce((acc, elem) => (acc || elem), false);
+  }
+
+  // replaces number words with the correct number, to standardize answer format
+  static replaceNumber(word){
```

Comment:
I'm a little skeptical about this function because it's subject to only a range of numbers and doesn't cover edge cases what if the number is 17 or anything else? 

Could you look into converting this into generic functions where you could convert an input (say an ordinal to a number) or (roman numeral to number?)